.gitignore

```diff
@@ -2,3 +2,4 @@
 /composer.lock
 /phpunit.xml
 /vendor/
+.idea/
```

Comment:
Definitely no go for this, you may want to read about [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
@kubawerlos I'll remove it. Thanks!